### ISSUE TITLE
Opening a new window causes site to request login credentials again

### ISSUE THREAD

**Issue by GSI** [uzbl/uzbl#350]
Please paste the output of `uzbl-core --bug-info` here:

```
Commit: v0.9.1-21-g73a126bf
GTK compile: 3.22.7
GTK run: 3.22.7
WebKit compile: 2.4.11
WebKit run: 2.4.11
WebKit2: 0
libsoup compile: 2.56.0
libsoup run: 2.56.0
```

> Is there a specific website which shows the problem?

http://www.airbnb.com and in **redmine**. Notably both are **rails** based.

Cross-tested in Chrome, where this doesn't happen.

**Comment by mathstuf** [comment]
Do you have an on-disk store for session cookies? If not, only live instances will share them and new ones will get nothing

**Comment by GSI** [comment]
> Do you have an on-disk store for session cookies?

Yes. Otherwise the error would happen on any website, I suppose (?)
